content/doc/book/installing/docker-compose.adoc

```diff
@@ -0,0 +1,70 @@
+---
+layout: section
+title: Docker Compose
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:toclevels: 4
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
++
+*Note:* This post refers to link:/doc/installing/docker[Installing Jenkins with Docker].
++
+
+Docker link:https://docs.docker.com/compose/[Compose] is a tool for defining and running multi-container Docker applications. With Compose, you use a YAML file to configure your application's services. Then, with a single command, you create and start all the services from your configuration. 
+
+Using Compose is basically a three-step process:
+
+. Define your app's environment with a `Dockerfile` so it can be reproduced anywhere.
+
+. Define the services that make up your app in `docker-compose.yml` so they can be run together in an isolated environment.
+
+. Run `docker compose up` and the https://docs.docker.com/compose/cli-command/[Docker compose command] starts and runs your entire app. You can alternatively run `docker-compose up` using the docker-compose binary.
+
+A `docker-compose.yml` looks like this:
+
+[source,yaml]
+----
+version: "3.9"  # optional since v1.27.0
+services:
+  web:
+    build: .
+    ports:
+      - "5000:5000"
+    volumes:
+      - .:/code
+      - logvolume01:/var/log
+    links:
+      - redis
+  redis:
+    image: redis
+volumes:
+  logvolume01: {}
+----
+
+For more information about the Compose file, see the https://docs.docker.com/compose/compose-file/[Compose file reference].
+
+Compose has commands for managing the whole lifecycle of your application:
+
+- Start, stop, and rebuild services
+- View the status of running services
+- Stream the log output of running services
+- Run a one-off command on a service
+
+=== Install Docker Compose
+
+To install Docker on your operating system, follow documentation of the
+link:https://docs.docker.com/compose/install/[Install Docker Compose]
```

Comment:
```suggestion
To install Docker on your operating system, follow the
link:https://docs.docker.com/compose/install/[Install Docker Compose] documentation.
```

Reply:
We certainly want to be careful about muddying up the Jenkins docs with a lot of details about other technologies but this piece does a nice job of providing a simple overview of the pieces required then references the Docker documentation for details.  This could help users who are fairly new to Docker and need to see the "big picture" for how to use it with Jenkins.